contributor_docs/zh/contributor_guidelines.md

```diff
@@ -1,204 +1,353 @@
+
+该模板有以下字段需要填写：
+
+1. _p5.js 中与之最相关的子领域是？_ - 回答该问题将触发自动标记功能，为问题打上相关的[标签](https://github.com/processing/p5.js/blob/main/contributor_docs/issue_labels.md)，这有助于我们更好地识别和回应你的问题。
+2. _p5.js 版本_ - 你可以在`<script>`标签的链接中或者在 p5.js/p5.min.js 文件的第一行找到 p5.js 的版本号。它的格式类似于`1.4.2`（由三个句点分隔的数字）。
+3. _Web 浏览器及版本_ - 该信息有助于我们区分代码在不同浏览器中的行为差异。浏览器版本号可以根据下方表格中对应不同浏览器的步骤找到。
+
+<table>
+
+<tr>
+
+<td>
+
+Chrome
+
+</td>
+
+<td>
+
+Firefox
+
+</td>
+
+<td>
+
+Safari
 
-### "发现了一个错误"
-当你在使用p5.js时遇到可能的错误行为或某些行为与文档描述不符时，请使用以下模板。请注意，如果你尝试调试自己的代码或弄清楚为什么你的草图的行为与你预期的不同，并且你认为可能是你的代码的问题，你应该在[论坛](https://discourse.processing.org)上提问。如果后来确定你的问题确实源自p5.js，你可以随时打开一个问题并使用这个模板。
+</td>
 
-这个模板有几个需要填写的字段：
-1. "p5.js 中最相关的子领域是什么？" - 这有助于相关的管理者识别和回应你的问题。这将自动使用相关的[标签](./issue_labels.md)给问题打上标签。
-2. "p5.js 版本" - 你可以在`<script>`标签的链接中或者在p5.js/p5.min.js文件的第一行找到p5.js的版本号。它的格式类似于`1.4.2`（由三个点分隔的数字）。
-3. "Web浏览器及版本" - 在Chrome中，在地址栏中输入"chrome://version"；在Firefox中输入"about:support"；在Safari中，在顶部栏"Safari"菜单项下选择"关于Safari"。这有助于我们区分不同浏览器之间的行为差异。
-4. "操作系统" - 如果可能，请提供操作系统的版本号，例如`macOS 12.5`。有些错误可能也源自操作系统的行为。
-5. "重现步骤" - 这可能是最重要的信息。你应该列出详细的步骤来重现你所遇到的错误。分享一个能够演示问题的基本示例代码可以让其他人更容易重现你面临的错误并开始制定解决方案。
+</tr>
 
-正如上面提到的，模板中的许多不同字段旨在重现错误。你提供的关于草图环境以及其他人如何重现你所看到的问题的更多信息，将更容易让任何人理解你的问题并开始寻找解决方案。**请尽可能提供详细的信息，避免使用泛泛的陈述**，例如不要说"image()函数不起作用"，而是要更具体地描述，例如"image()函数无法以正确的尺寸显示加载的GIF图像"。描述你所面临的错误的一种有用方式是描述两个方面：1. 你期望你分享的示例代码执行什么样的行为（预期行为）；2. 示例代码实际上做了什么（实际行为）。
+<tr>
 
-如果您希望为您刚刚报告的错误贡献修复代码，您可以在描述中说明。如果您可以提供一个简单的建议来修复您所描述的错误，那对于问题评审者来说将非常有帮助，因为他们需要知道您需要多少支持来贡献修复代码。在开始处理拉取请求之前，至少需要一个领域监督人或维护者批准修复该错误。在问题获得修复批准之前，任何在此之前提交的拉取请求都将被关闭。**您不应该在没有相应问题或在问题获得实施批准之前提交拉取请求（或开始进行代码更改）**，这是因为无法保证您的建议将被接受，而您所做的工作最终可能不会被合并。
+<td>
 
-### "现有功能增强"
-如果您希望修改或添加p5.js的现有功能（函数、常量、渲染等），应使用此模板。例如，如果您想为`color()`函数和其他接受颜色的函数添加一种新的定义颜色的方式，这就是要使用的模板。
+在地址栏中输入 `chrome://version`
 
-该模板有几个字段需要填写：
-1. "提高可访问性" - 这是一个非可选字段，您需要在此处说明添加建议的功能增强将如何帮助p5.js [提高可访问性](./access.md)，使在创意艺术或技术领域中历史上被边缘化的人能够更好地使用。**如果没有提供此项内容，将不会接受任何提案**，尽管您可以填写"不确定"并邀请社区的其他成员提供关于如何提高p5.js可访问性的论点。
-2. "p5.js中最适合的子领域是什么？" - 这将帮助相关的管理员确定并回应您的问题。这将自动使用相关的[标签](./issue_labels.md)标记该问题。
+</td>
+
+<td>
+
+在地址栏中输入   `about:support`
+
+</td>
+
+<td>
+
+打开顶部 “Safari” 菜单, 选择 “关于 Safari”
+
+</td>
+
+</tr>
+
+</table>
+
+4. _操作系统_ - 如可能，请提供操作系统的版本号，例如`macOS 12.5`。某些错误也可能源于操作系统。
+5. _重现步骤_ - 这可能是最重要的信息。请详细列出重现你所遇到的错误的步骤。贴出能够展示问题产生的简单示例代码可以让其他人更容易重现你的错误并制定解决方案。
+
+**重现错误是最关键的！**该模板中的许多字段都旨在能够重现 Bug。你提供的关于草图环境的信息越多、重现问题的步骤越丰富，别人也就越容易理解你的问题并探索解决方案。
+
+**请尽可能提供详细的信息，避免使用泛泛的陈述**。例如，不要说"image()函数不好用了"，而要说得更具体，比如："image()函数无法以正确的尺寸显示加载的 GIF 图像"。描述以下两个方面可以使你的问题描述更加清晰：
+
+1. 你期望你分享的示例代码执行什么样的行为（预期行为）；
+2. 示例代码实际上做了什么（实际行为）。
+
+如果您希望为您刚刚报告的错误贡献修复代码，您可以在描述中说明。如果您可以提供一个简单的建议来修复您所描述的错误，那对于问题评审者来说将非常有帮助，因为他们需要知道您需要多少支持来贡献修复代码。
+
+**您不应该在没有相应问题或在问题获得实施批准之前提交拉取请求（或开始进行代码更改）**，这是因为无法保证您的建议将被接受，而您所做的工作最终可能不会被合并。
```

Comment:
原文：that is because the proposed fix may not be accepted, need a different approach entirely, or the actual problem is somewhere else. Any pull requests filed before the issue has been approved for fixing will be closed until approval is given to the issue.

这里的翻译漏了一些原文细节，请补上